molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/static/css/_admin_css.scss

```diff
@@ -1,5 +1,8 @@
 .versions{
   color: black;
+  a {
+    color: red;
+  }
```

Comment:
Could you explain why this is necessary? If it is, would it be better to go with the [reds wagtail uses](https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailadmin/static_src/wagtailadmin/scss/_variables.scss#L32-L36) for consistency?